File: CS50/week_6/problem_set_6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if 1 <= height <= 8:
        break

for i in range(height):
    # Print spaces
    print(" " * (height - i - 1), end="")

    # Print the first set of hashes
    print("#" * (i + 1), end="")

    # Print the gap
    print("  ", end="")

    # Print the second set of hashes
    print("#" * (i + 1))

File: CS50/week_1/problem_set_1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // get height from user
    int input;
    do
    {
        input = get_int("Height:");
    }
    while (input <= 0 || input > 8);

    // i = height of pyramid
    int height = input;
    // first row of spaces
    int space1;
    // first row of hashes
    int hash1;
    //second row of spaces
    int space2;
    // second row of hashes
    int hash2;


    for (height = input; height > 0; height--)
    {
        for (space1 = 1; space1 < height; space1++)
        {
            printf(" ");
        }
        for (hash1 = (space1 - 1); hash1 < input; hash1++)
        {
            printf("#");
        }
        for (space2 = 2; space2 > 0; space2--)
        {
            printf(" ");
        }
        for (hash2 = (space1 - 1); hash2 < input; hash2++)
        {
            printf("#");
        }
        printf("\n");
    }


}